web/blog/2021-12-02-waspello.md

```diff
@@ -19,6 +16,10 @@ import ImgWithCaption from './components/ImgWithCaption'
     <Link to={'https://waspello.netlify.app/'}>Try Waspello here!</Link> | <Link to={'https://github.com/wasp-lang/wasp/blob/main/examples/waspello/main.wasp'}>See the code</Link>
 </p>
 
+We've built a Trello clone using Wasp!
```

Comment:
@matijaSos I put this one in as a placeholder. Should we come up with something more engaging?

Reply:
```suggestion
We've built a Trello clone using Wasp! Read on to learn how it went and how you can contribute. 
```

Reply:
This could definitely be even better but just added this off the top of the head :D